extensions/flac/proguard-rules.txt

```diff
@@ -12,6 +12,6 @@
 -keep class com.google.android.exoplayer2.extractor.FlacStreamMetadata {
     *;
 }
--keep class com.google.android.exoplayer2.metadata.flac.PictureFrame {
+-keep class com.google.android.exoplayer2.metadata.vorbis.PictureFrame {
```

Comment:
The comment at the top of this section suggests you also need to change lines like this:
https://github.com/google/ExoPlayer/blob/b3981be8b91992976d78c8803349db9680b648a7/extensions/flac/src/main/jni/flac_jni.cc#L126

Reply:
Resolved with c2d0649. I grepped for any instance of "(flac.PictureFrame)|(flac.VorbisComment)" and could only find documentation and the class definitions themselves.